Dungeons/Dungeons/CorridorNodeLayouter.cs

```diff
@@ -96,39 +96,59 @@ protected virtual void LayoutNodes(DungeonNode level, List<DungeonNode> mazeNode
       var d = new DungeonGenerator(container);
       var infoC = new GenerationInfo();
       infoC.NumberOfRooms = 1;
+      var numberOfNodes = mazeNodes.Count;
 
       mazeNodes[0].Placement = RoomPlacement.LeftUpper;
-      mazeNodes[1].Placement = RoomPlacement.RightUpper;
-      mazeNodes[2].Placement = RoomPlacement.RightLower;
-      mazeNodes[3].Placement = RoomPlacement.LeftLower;
-      mazeNodes[4].Placement = RoomPlacement.Center;
-
       mazeNodes[0].GenerateDoors(RoomPlacement.LeftUpper);
-      mazeNodes[1].GenerateDoors(RoomPlacement.LeftLower);
-      mazeNodes[2].GenerateDoors(RoomPlacement.RightUpper);
-      mazeNodes[3].GenerateDoors(RoomPlacement.RightLower);
-      mazeNodes[4].GenerateDoors(RoomPlacement.Center);
+      if (mazeNodes.Count > 1)
```

Comment:
Jak już mamy enumy, zamiast ifować chyba można by:
var placementToDoor = new Dictionary<RoomPlacement, RoomPlacement>
      { 
        { RoomPlacement.LeftUpper, RoomPlacement.LeftUpper},
        { RoomPlacement.RightUpper, RoomPlacement.LeftLower}
  ....
      };

      for (int i = 0; i < mazeNodes.Count; i++)
      {
        var placement = (RoomPlacement)(i);
        mazeNodes[i].Placement = placement;
        mazeNodes[i].GenerateDoors(placementToDoor[placement]);
      }